meta-resin-common/recipes-connectivity/modemmanager/modemmanager_1.6.8.bb

```diff
@@ -48,3 +48,8 @@ FILES_${PN}-staticdev += " \
 FILES_${PN}-dbg += "${libdir}/ModemManager/.debug"
 
 SYSTEMD_SERVICE_${PN} = "ModemManager.service"
+
+# XXX
+# Introspection is not available in all yocto versions. We don't take advantage of
+# it in resin so disabling it for the time being. To be revised.
+EXTRA_OECONF_prepend = "--disable-introspection "
```

Comment:
Can this be handled with PACKAGECONFIG? Such as we do for NetworkManager: https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%25.bbappend#L24

Reply:
Yes it can. Do you see any advantage in switching to that or you just mentioned as an option?

Reply:
Just as an option, to be in-line with the official way Yocto provides for using configure switch options.